lua/astrocommunity/pack/cpp/init.lua

```diff
@@ -46,6 +46,14 @@ return {
                 if assert(vim.lsp.get_client_by_id(args.data.client_id)).name == "clangd" then
                   require "clangd_extensions"
                   vim.api.nvim_del_augroup_by_name "clangd_extensions"
+                  require("astrocore").set_mappings({
+                    n = {
+                      ["<Leader>lw"] = {
+                        "<cmd>ClangdSwitchSourceHeader<CR>",
+                        desc = "Clangd: Switch source/header file",
+                      },
+                    },
+                  }, {})
```

Comment:
```suggestion
                  }, { buffer = args.buf })
```

Only set mappings for buffer that clangd is attached to

Reply:
actually this isn't a super great method because this will only work for the first language server. one sec